README.md

```diff
@@ -54,18 +54,18 @@ import ReactMorph from "react-morph";
 * Simplicity
 * No hardcoded absolute positions
 * All GPU accelerated props
-* No Layout or Paint browser rendering
+* No layout or paint browser rendering
 
-## Take in mind
+## Keep in mind
 
 Caveats:
 
-* You need to remove extra white spaces, to match the real element's width, the solutions is `display: inline-block` or a wrapping element to the content.
-* Margins it's always tricky because create extra white space, you could either wrap the content in another element and animate it or be sure to match both state margins.
+* You need to remove extra whitespace, to match the real element's width, the solution is `display: inline-block` or a wrapping element to the content.
+* Margins are always tricky because they create extra white space. You cam either wrap the content in another element and animate it or be sure to match both the margins in both states.
```

Comment:
Thanks a lot! I think you mean `can` Instead of `cam`.

Reply:
Whoops! Yes of course. Fixed :)